bbc/simorgh
# Navigation changes to add Euro 2024 and update redirected links

Resolves JIRA [number]

Overall changes
======
- Added Euro 2024 topic links to the navgation bar of Serbian and Turkce
- Updated links to redirected FIXs to spare users the redirect on Arabic and Ukrainian


Code changes
======

- Edited Navigation section of src/app/lib/config/services/serbian.ts to add Euro 2024 topic link in second position
- Edited Navigation section of src/app/lib/config/services/turkce.ts to replace March election link with Euro 2024
- Edited Navigation section of src/app/lib/config/services/arabic.ts to update last two items wiht topic links
- Edited Navigation section of src/app/lib/config/services/ukrainain.ts to update second to last item with topic link

Testing
======
1. Open Serbian Lat and Cyr front pages, second link in the nav should be Euro 2024 and open the Euro 2024 topic
2. Open Turkce's front page, third link in the nav should be Euro 2024 and open the Euro 2024 topic
3. Open Arabic front page, last two items on the nav should point and Open  aTipo topic not a CPS Fix
4. Open Ukrainian's front page, second to last items on the nav should point and open a Tipo topic not a CPS Fix

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
